virtcontainers/capabilities.go

```diff
@@ -47,3 +49,25 @@ func (caps *capabilities) isMultiQueueSupported() bool {
 func (caps *capabilities) setMultiQueueSupport() {
 	caps.flags |= multiQueueSupport
 }
+
+func (caps *capabilities) isHotplugSupported() bool {
+	if caps.flags&hotplugUnsupported == 0 {
+		return true
+	}
+	return false
+}
+
+func (caps *capabilities) setHotplugUnsupported() {
+	caps.flags |= hotplugUnsupported
+}
+
+func (caps *capabilities) is9pSupported() bool {
```

Comment:
@egernst instead of calling this is9pSupported, should this not be something more generic. 9p is one way to share files between host and the VM?

Reply:
Yeah we could call it `filesystemSharingSupport`. @egernst @devimc ?